addons/marketing_campaign/tests/test_marketing_campaign_flow.py

```diff
@@ -12,14 +12,14 @@ def test_00_marketing_campaign_flow(self):
 
         # In order to test process of compaign, I start compaign.
         partner_channel = self.env.ref('marketing_campaign.marketing_campaign_openerppartnerchannel')
-        partner_channel.signal_workflow('state_running_set')
+        partner_channel.state_running_set()
```

Comment:
Sure, though marketing campaign is being rewritten from scratch and going to enterprise ( https://www.odoo.com/web#id=9314&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720 ), so this folder will not live for long.
The diff is much smaller if the methods are not renamed as this workflow had 1 method per workflow change.